va_explorer/templates/base.html

```diff
@@ -19,7 +19,6 @@
     {% block css %}
     <link href="{% static 'vendor/bootstrap-4.3.1/bootstrap.css' %}" rel="stylesheet">
     <link href="{% static 'vendor/select2-4.1.0-beta.1/select2.css' %}" rel="stylesheet">
-
```

Comment:
Should this include the fab.js and fab.css?

Do we need to include _fab_nav.html in base.html as well?

Reply:
sorry, the fab files weren't part of this pull request. They accidentally got carried over when I switched branches after reviewing @ajbarnes's navigation PR. Will delete them from my PR